### PR TITLE
Create code review comment and wire up to front-end

### PR DESCRIPTION
**Pull request by maureensturgeon**
https://user-images.githubusercontent.com/24235215/169138877-675e404a-ecf3-4f5d-84d9-fbab9a143e69.mov

## Links
- jira ticket: [LP-2307](https://codedotorg.atlassian.net/browse/LP-2307)

## Testing story
Tested locally and added to my later testing JIRA ticket https://codedotorg.atlassian.net/browse/LP-2376

## Follow-up work
Once Jame's PR is merged in with the project active record model, we'll want to do validation that the code review comment can be created

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by molly-moen — APPROVED** [review]

File: dashboard/app/controllers/code_reviews_controller.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@ def create
     authorize! :create, code_review, project_owner_id
     code_review.save!
 
-    render json: code_review.summarize
+    render json: code_review.summarize_with_comments
```
Comment (jamescodeorg) [inline dashboard/app/controllers/code_reviews_controller.rb:49]: Why this change and the one below?

Comment (maureensturgeon) [reply dashboard/app/controllers/code_reviews_controller.rb:49]: I use the response to set to the state of the component. It would be possible to just modify the state locally for the changes that are made to the record, but this seemed more straight-forward

**Review by jamescodeorg — APPROVED** [review]
